Documentation/security/network/encryption-ipsec.rst

```diff
@@ -171,6 +171,31 @@ commands:
        15:16:21.627699 IP 10.60.1.1 > 10.60.0.1: ESP(spi=0x00000001,seq=0x57e4), length 100
        15:16:21.628408 IP 10.60.1.1 > 10.60.0.1: ESP(spi=0x00000001,seq=0x57e5), length 100
 
+.. note::
```

Comment:
I'm thinking it might make sense to add a new section, "Monitoring", instead of a note in "Validate the Setup" section. See also [Cilium docs guidelines](https://docs.cilium.io/en/stable/contributing/docs/docsstyle/#callouts). What do you think?

Reply:
Yep, that's definitively our case, I missed the callouts part :sweat_smile: 
Thanks for pointing out, refactoring it in a while!